### ISSUE TITLE
__toString in sidebar

### ISSUE THREAD

**Issue by mhd-yasser-haddad** [thedevdojo/voyager#4836]
## Version information
 - Laravel: v6.2.^
 - Voyager: v1.3.2
 - PHP: 7.2
 - Database: MySql

## Description

When I uploaded my project to the server everything was fine until I logged in to the admin panel
Where I faced this error related to the side bar

## Steps To Reproduce
1- uploaded the project to the server
2- uploaded the database
3- logged into the admin panel

## Expected behavior
The sidebar to work properly

![123](https://user-images.githubusercontent.com/44609144/77324779-efa1e600-6d1f-11ea-8f33-6646dee0ebe0.PNG)

**Comment by MrCrayon** [comment]
Please provide exact version for Voyager and line where that error is happening.

**Comment by mhd-yasser-haddad** [comment]
@MrCrayon  I hope you find everything you asked for in the following pics:

The exact line:
![234](https://user-images.githubusercontent.com/44609144/77326124-034e4c00-6d22-11ea-8500-3caf76d2b094.PNG)

The version:
![345](https://user-images.githubusercontent.com/44609144/77326144-0ba68700-6d22-11ea-9dfc-eec0dc7b6d3d.PNG)


Note that when I remove the sidebar from the master layout everything works properly

**Comment by MrCrayon** [comment]
> @MrCrayon I hope you find everything you asked for in the following pics:

Not even close :grinning: 

You are requiring Voyager from 1.3 and up, valid versions are 1.3.0, 1.3.1 and 1.3.2 you need to check which one you have installed.
You can see it in Voyager footer
![Schermata da 2020-03-23 15-33-22](https://user-images.githubusercontent.com/6875243/77327549-b0719600-6d1b-11ea-9d65-cae2da02b820.png)
Or you ca try from command line:
```shell
composer show tcg/voyager | grep versions
```

For the error we need to know the line in sidebar.blade.php where the error is happening, follow the Stack trace.

**Comment by mhd-yasser-haddad** [comment]
@MrCrayon Here is the things you asked for:
Stack Trace:
![stack trace](https://user-images.githubusercontent.com/44609144/77328877-d734ca00-6d25-11ea-9d03-21eca1f4d9f9.PNG)

Voyager Version:
![version](https://user-images.githubusercontent.com/44609144/77328895-df8d0500-6d25-11ea-93d7-80f26e070c7a.PNG)

Note that in the localhost the problem doesn't exist only when I uploaded the project to the server

**Comment by MrCrayon** [comment]
Quite strange, something might be happening here:
https://github.com/the-control-group/voyager/blob/643b02b23f7758bbc3c1385dd6c919ec573e201e/src/Models/Menu.php#L53

Try to clear all your caches and if that doesn't work try to debug inside that method.

**Comment by mhd-yasser-haddad** [comment]
@MrCrayon I tried clearing the cache of the project but it didn't work, can you tell me how can I debug the program?

**Comment by mhd-yasser-haddad** [comment]
@MrCrayon and everyone who is facing this problem, this is how I solved it:
1- I removed the side bar from the master layout so that the view could work.
(I removed it by going to tcg>voyager>resources>views>master.blade.php and removing the "@include('voyager::dashboard.sidebar')" line from the view)

2- I headed to the menu builder to see the menu and edit it by following this link (admin/menus/1/builder)

3- When the menu builder opened some of the items were looking like this:
![menu builder](https://user-images.githubusercontent.com/44609144/77355781-c304c300-6d4d-11ea-97e4-4c5ea227af22.PNG)
The menu item you are seeing in the picture is supposed to be the dashboard (home page) but for some reason it looks like this.
I tried to edit it but nothing worked, also I checked the database to see if anything is missing but everything was fine and all the data was there.

4- I replaced the empty menu items with new ones by adding new menu items and deleted the empty ones

5- And added the sidebar again in the master layout and everything worked properly

That's all.
